Contracts/TokenAuth Api/response.json

```diff
@@ -0,0 +1,11 @@
+//Response Data with 200:ok 
+{
+    //Mandatory Validity of the Token
+    "AuthenticationStatusCode" : "200",
+
+    //Mandatory String of message
+    "Message":[
```

Comment:
if in case of valid response , api consumer has no use of this field. Please do not pass redundant info in api response.